src/sentry/api/serializers/models/exporteddata.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+from sentry.api.serializers import Serializer, serialize, register
+from sentry.models import ExportedData, User
+
+
+@register(ExportedData)
+class ExportedDataSerializer(Serializer):
+    def get_attrs(self, item_list, user, **kwagrs):
+        attrs = {}
+        for item in item_list:
+            user = User.objects.get(id=item.user.id)
```

Comment:
if we can use `item.user_id` here we should to save an extra query (iirc)

Reply:
done!

Reply:
Oh man now im gettting in my own head, is there any reason why I should even be using `User.objects.get(` instead of just reading `item.user.id, item.user.username, and item.user.email` directly from `item`? I'm confusing myself with code I wrote literally yesterday